### PR TITLE
fix: auth success message printer fix

### PR DESCRIPTION
**Pull request by akshbhu**
#### Description of changes
- fixed success message prompts when auth  resource is added

#### Description of how you validated changes
- manual tested

#### Checklist

- [x] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by jhockett — APPROVED** [review]

File: packages/amplify-category-auth/src/provider-utils/awscloudformation/utils/message-printer.ts [hunk 1]
```diff
@@ -6,26 +6,24 @@ import { printer } from 'amplify-prompts';
...
+  if (!skipNextSteps) {
+    printCommonText();
+  }
+};
 
 /**
  * A factory function that returns a function that prints the "success message" after updating auth
  * @param context The amplify print object
  */
 export const getPostUpdateAuthMessagePrinter = (print: any) => (resourceName: string) => {
```
Comment (edwardfoyle) [inline packages/amplify-category-auth/src/provider-utils/awscloudformation/utils/message-printer.ts:21]: nit: don't need to pass in `print` anymore

File: packages/amplify-category-auth/src/provider-utils/awscloudformation/utils/message-printer.ts [hunk 2]
```diff
@@ -6,26 +6,24 @@ import { printer } from 'amplify-prompts';
  * A factory function that returns a function that prints the "success message" after adding auth
  * @param print The amplify print object
  */
-export const getPostAddAuthMessagePrinter =
-  (print: any) =>
-  (resourceName: string, skipNextSteps: boolean = false) => {
-    print.success(`Successfully added auth resource ${resourceName} locally`);
+export const getPostAddAuthMessagePrinter = (resourceName: string, skipNextSteps: boolean = false) => {
```
Comment (edwardfoyle) [inline packages/amplify-category-auth/src/provider-utils/awscloudformation/utils/message-printer.ts:9]: won't this update require a corresponding update anywhere `getPostAddAuthMessagePrinter` is used?

Comment (akshbhu) [reply packages/amplify-category-auth/src/provider-utils/awscloudformation/utils/message-printer.ts:9]: this only gets called here

**Comment by akshbhu** [comment]
closing in favour of https://github.com/aws-amplify/amplify-cli/pull/8827

**Pull request closed without merging**